pkg/domain/infra/abi/containers.go

```diff
@@ -1539,8 +1539,7 @@ func (ic *ContainerEngine) ContainerStats(ctx context.Context, namesOrIds []stri
 		}
 	case len(namesOrIds) > 0:
 		containerFunc = func() ([]*libpod.Container, error) { return ic.Libpod.GetContainersByList(namesOrIds) }
-	default:
-		// No containers, no latest -> query all!
+	case options.All:
```

Comment:
We still need the default case where we only get the running containers as described in the docs by default it should only show running containers.

Reply:
@Luap99 If you scroll up from here, you will see that 
```go
	containerFunc := ic.Libpod.GetRunningContainers
	queryAll := false
```
https://github.com/containers/podman/pull/19254/files#diff-025657b44bdf62ef5473f82e0b2c2a7580629db73a1b61d02ddcd6be48a4b000L1529-L1531

But if it's more readable I can change it to:
```go
	var containerFunc func() ([]*libpod.Container, error)
	var queryAll bool
       switch {
         ...
         default:
           containerFunc = ic.Libpod.GetRunningContainers
           queryAll = true
```

Reply:
Ah correct I missed that, I think changing it to that would be more readable.